pkg/systemd/services/services.jsx

```diff
@@ -921,8 +756,10 @@
         const type = getFilterLabelKey(typeLabel);
 
         if (type) {
+            debug("onDeleteChip: type", type, "unchanged filters");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18422-20230307-104236-77724996-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)